cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/config/BrokerCapacityInfo.java

```diff
@@ -119,7 +119,7 @@ public BrokerCapacityInfo(Map<Resource, Double> capacity) {
   }
 
   /**
-   * @return True if the capacity of the broker for at least one resource is based on an estimation, false otherwise.
+   * @return True if the capacity of the broker for at least one resource is based on an estimation, {@code false} otherwise.
```

Comment:
```suggestion
   * @return {@code true} if the capacity of the broker for at least one resource is based on an estimation, {@code false} otherwise.
```